src/Npgsql/NpgsqlParameter.cs

```diff
@@ -503,6 +505,11 @@ public sealed override int Size
         public NpgsqlParameterCollection Collection { get; set; }
 #pragma warning restore CA2227
 
+        /// <summary>
+        /// The PostgreSQL data type as discovered from pg_type
```

Comment:
Maybe make this more explicit, explaining derivation in a few words and referencing CommandBuilder with `<see .../>` etc. `pg_type` probably doesn't mean a lot to most people.

Reply:
Extended the comment as suggested.